src/listener/utils/resource-transforms.js

```diff
@@ -49,6 +49,10 @@ function addPeriod(text) {
   return text;
 }
 
+function isVowel(x) {
```

Comment:
line 162 of this file, it was being called but wasn't actually a method anywhere